src/components/UnsuportedContent.vue

```diff
@@ -65,7 +57,7 @@ export default {
       default: 'Falha ao enviar a mensagem'
```

Comment:
Is this text right? Or is this only a default value that is replaced by the text on the Blip user Language?